docs/tables/aws_ec2_gateway_load_balancer.md

```diff
@@ -0,0 +1,101 @@
+# Table: aws_ec2_gateway_load_balancer
+
+Gateway Load Balancer makes it easy to deploy, scale, and run third-party virtual networking appliances. Providing load balancing and auto scaling for fleets of third-party appliances, Gateway Load Balancer is transparent to the source and destination of traffic. This capability makes it well suited for working with third-party appliances for security, network analytics, and other use cases
+
+## Examples
+
+### Basic gateway load balancer  info
+
+```sql
+select
+  name,
+  arn,
+  type,
+  state_code,
+  vpc_id,
+  availability_zones
+from
+  aws_ec2_gateway_load_balancer
+
+```
+
+### Availability zone information of all the gateway load balancer
+
+```sql
+select
+  name,
+  az ->> 'LoadBalancerAddresses' as load_balancer_addresses,
+  az ->> 'OutpostId' as outpost_id,
+  az ->> 'SubnetId' as subnet_id,
+  az ->> 'ZoneName' as zone_name
+from
+  aws_ec2_gateway_load_balancer
+  cross join jsonb_array_elements(availability_zones) as az;
```

Comment:
Avoid using cross join when you don't need it in the examples.
```suggestion
select
  name,
  az ->> 'LoadBalancerAddresses' as load_balancer_addresses,
  az ->> 'OutpostId' as outpost_id,
  az ->> 'SubnetId' as subnet_id,
  az ->> 'ZoneName' as zone_name
from
  aws_ec2_gateway_load_balancer,
  jsonb_array_elements(availability_zones) as az;
```
This will do the same